NEWS.md

```diff
@@ -376,6 +376,8 @@
 
 44. In v1.13.2 a version of an old bug was reintroduced where during a grouping operation list columns could retain a pointer to the last group. This affected only attributes of list elements and only if those were updated during the grouping operation, [#4963](https://github.com/Rdatatable/data.table/issues/4963). Thanks to @fujiaxiang for reporting and @avimallu and Václav Tlapák for investigating and the PR.
 
+45. `shift(x, fill)` could error if `x` had type `bit64::integer64` and a `fill` argument was also provided, [#4865](https://github.com/Rdatatable/data.table/issues/4865). Thanks to @peterlittlejohn for reporting and Benjamin Schwendinger for the PR.
```

Comment:
Does  `could` really mean `would` here? If so, maybe reword:

> `shift(x, fill)` no longer fails when `x` is `integer64` and `fill` is provided

`could` makes it seem like sometimes fails, sometimes doesn't, in which case we'd want to explicate better what those failure cases are (and are not)

Reply:
In this case its really a `could`, since it worked out `x` being `bit64::integer64` and `fill` being `integer32`. I tweaked the message a bit, but I'm not sure if `integer32` is an appropriate wording for NEWS.